shuup/core/management/commands/shuup_migrate_encrypted_field.py

```diff
@@ -0,0 +1,57 @@
+# This file is part of Shuup.
+#
+# Copyright (c) 2012-2021, Shuup Commerce Inc. All rights reserved.
+#
+# This source code is licensed under the OSL-3.0 license found in the
+# LICENSE file in the root directory of this source tree.
+from django.core.management.base import BaseCommand
+
+from .migrator import MirageMigrator as Migrator
+
+
+class Command(BaseCommand):
```

Comment:
Same here, it handles the parameter idfield and uses MirageMigrator instead of mirage.migration.Migrator.